datalad/distribution/get.py

```diff
@@ -96,6 +97,11 @@ class Get(Interface):
             constraints=EnsureStr() | EnsureNone()),
         recursive=recursion_flag,
         recursion_limit=recursion_limit,
+        fulfill=Parameter(
+            args=("--fulfill",),
+            choices=('auto', 'all'),
+            doc="""whether to fulfill file handles. The default will do the
```

Comment:
I have no clue what you are talking about about and what fulfill means especially in respect to 'get' -- 'get ' is to get the bloody handles, there  should be no need for additional option